stacker_blueprints/empire/daemon.py

```diff
@@ -0,0 +1,604 @@
+            ),
+            "default": ""},
+        "TokenSecret": {
+            "type": "String",
+            "description": "EMPIRE_TOKEN_SECRET",
+            "default": ""},
+        "TugboatUrl": {
+            "type": "String",
+            "description": "EMPIRE_TUGBOAT_URL",
+            "default": ""},
+        "ConveyorUrl": {
+            "type": "String",
+            "description": "EMPIRE_CONVEYOR_URL",
+            "default": ""},
+        "LogsStreamer": {
+            "type": "String",
+            "description": "EMPIRE_LOGS_STREAMER",
+            "default": ""},
+        "Reporter": {
+            "type": "String",
+            "description": "The reporter to use to report errors",
+            "default": ""},
+        "InternalZoneId": {
+            "type": "String",
+            "description": "The ID for the route53 zone for internal DNS"},
+        "PrivateAppELBSG": {
+            "type": "String",
+            "description": (
+                "Security group to attach to internal load balancers"
+            ),
+            "default": ""},
+        "PublicAppELBSG": {
+            "type": "String",
+            "description": "Security group to attach to public load balancers",
+            "default": ""},
+        "MinionCluster": {
+            "type": "String",
+            "description": "ECS Cluster for the Minions.",
+            "default": ""},
+        "ControllerCluster": {
+            "type": "String",
+            "description": "ECS Cluster for the Controllers.",
+            "default": ""},
+        "RunLogsBackend": {
+            "type": "String",
+            "allowed_values": ["cloudwatch", "stdout"],
+            "description": "The backend to use for empire run logs.",
+            "default": "stdout"},
+        "RunLogsCloudwatchGroup": {
+            "type": "String",
+            "description": (
+                "The cloudwatch log group to use for run logs if the"
+                " 'RunLogsBackend' is set to 'cloudwatch'. If not provided, one"
+                " will be created for the run logs backend."
+            ),
+            "default": ""},
+        "EventsBackend": {
+            "type": "String",
+            "allowed_values": ["sns", "stdout", ""],
+            "description": (
+                "The backend to use for empire events. If 'sns' is specified,"
+                " provide EventsSNSTopicName to use a specific topic, or else"
+                " one will be created for you."
+            ),
+            "default": "stdout"},
+        "EventsSNSTopicName": {
+            "type": "String",
+            "description": (
+                "The SNS topic to use if the 'EventsBackend' is set to 'sns'."
+                " If not provided, one will be created for the sns backend."
+            ),
+            "default": ""},
+        "TaskMemory": {
+            "type": "Number",
+            "description": "The number of MiB to reserve for the task.",
```

Comment:
yes?

this is the memory specified in the task definition which gets used with each task?